### PR TITLE
link to the new 'contribute to SCS' page.

### PR DESCRIPTION
**Pull request by fkr**
As more infos are added to the new page, this section should be
removed complety and replaced just by one sentence.

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

File: _i18n/de/use.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ Projekt auf.
 
 ## Beitragen
 
+Informationen rund um unsere alltägliche Zusammenarbeit sind auf
+der Seite [zu SCS beitragen](https://scs.community/de/contribute/) zusammengefasst.
```
Comment (itrich) [inline _i18n/de/use.md:46]: I'd write "Zu SCS beitragen"

File: _i18n/en/use.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ project.
 
 ## Contributing
 
+Information regarding our daily workflow and meetings are
+summarized on the [contribute to SCS page](https://scs.community/contribute/).
```
Comment (itrich) [inline _i18n/en/use.md:41]: See above: "Contribute to SCS" page

**Review by itrich — CHANGES_REQUESTED** [review]

File: _i18n/de/use.md (hunk 1 above)
Comment (fkr) [reply _i18n/de/use.md:46]: am not really sure, I was wondering about that as well when I did the diff. If we write 'Zu' then the whole name should be in quotes. What do you think? Same applies then for the english variant imho.

Comment (itrich) [reply _i18n/de/use.md:46]: Yes, that's what I tried to point out. The complete line would be:

Informationen rund um unsere alltägliche Zusammenarbeit sind auf
der Seite ["Zu SCS beitragen"](https://scs.community/de/contribute/) zusammengefasst.

File: _i18n/en/use.md (hunk 2 above)
Comment (fkr) [reply _i18n/en/use.md]: ack. Thanks for the feedback!

**Review by itrich — APPROVED** [review]
Thank you!

**Pull request merged**